### DISCUSSION TITLE
v5.2.20 is causing my Pi 3b and Pi 4b to freeze with kernel 5.15

### DISCUSSION THREAD

**Discussion by glory86** [svpcom/wfb-ng#225] [Questions]
hello
i was working with v5.6.4.2
it had no issues at all but i wanted to maximize the transmitting power so i changed it to v5.2.20 
but unfortunately it is making my Pi 3b and Pi 4b to freeze instantaneously whenever i connect the wifi adapter
kernels used for Pi 3 and Pi 4 are 5.15.32-v7+ and 5.15.32-v7l+
there is no other 8812au driver installed

**Comment by svpcom** [comment]
Try this branch:  https://github.com/OpenHD/rtl8812au/tree/20201214.1-new

**Comment by glory86** [comment]
thanks for quick reply
unfortunately the solution didn't work because of ipx support is dropped for kernel 5.15
fatal error: net/ipx.h: No such file or directory

**Comment by svpcom** [comment]
Try prebuilt image for RPI3: https://github.com/svpcom/wifibroadcast/releases/download/wifibroadcast-22.05/wifibroadcast_22.05-rc2-rpi3.img.gz

**Comment by glory86** [comment]
> Try prebuilt image for RPI3: https://github.com/svpcom/wifibroadcast/releases/download/wifibroadcast-22.05/wifibroadcast_22.05-rc2-rpi3.img.gz

Thank you very much
It worked like a charm

Is there any installation guide on how you built this image?
including the driver, wifibroadcast, osd .... 
it's exactly what i wanted

**Comment by svpcom** [comment]
Yes, all steps are described in wiki:
https://github.com/svpcom/wifibroadcast/wiki/Setup-HOWTO
https://github.com/svpcom/wifibroadcast/blob/master/README.md

**Comment by glory86** [comment]
> Yes, all steps are described in wiki: https://github.com/svpcom/wifibroadcast/wiki/Setup-HOWTO https://github.com/svpcom/wifibroadcast/blob/master/README.md

thanks a lot
they are detailed as i wanted
i know this issue has turned into discussion but could you please tell me how to toggle OSD on and off for interacting with the PI

**Comment by svpcom** [comment]
`systemctl stop osd; systemctl disable osd` -- toggle off
`systemctl enable osd; systemctl start osd` -- toggle on

**Comment by glory86** [comment]
that's mean i have to access it through SSH to be able to do that 
i wish if there was a shortcut key to toggle osd visibility 
regards

**Comment by svpcom** [comment]
No, all interactions are via shell commands
